source/documentation/at-rules/forward.html.md.erb

```diff
@@ -1,7 +1,179 @@
 ---
 title: "@forward"
 introduction: >
-  TODO(nweiz): Write this
+  The `@forward` rule makes [mixins](mixin), [functions](function), and
+  [variables](../variables) from another Sass stylesheet available to
+  stylesheets that load your stylesheet with the [`@use` rule](use). It makes it
+  possible to organize Sass libraries across many files, while allowing their
+  users to load a single entrypoint file.
 ---
 
+It's written `@forward "<url>"`. It loads the module at the given URL just like
```

Comment:
"It's written" -> "The rule uses the syntax"

("It" here could be ambiguous, plus a more active voice, though it is longer)

Reply:
Changed to "The rule is written". I want to keep the word "written" because it's consistently used as a signal word for "here comes the general syntax".